cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinMapper.java

```diff
@@ -270,16 +272,24 @@ private Instant extractEntryLastModifiedDate() {
 
     private EntityDescription generateEntityDescription() {
         return new EntityDescription.Builder()
-                   .withLanguage(extractLanguage())
-                   .withMainTitle(extractMainTitle())
-                   .withPublicationDate(extractPublicationDate())
-                   .withReference(new ReferenceBuilder(cristinObject).buildReference())
-                   .withContributors(extractContributors())
-                   .withNpiSubjectHeading(extractNpiSubjectHeading())
-                   .withAbstract(extractAbstract())
-                   .withTags(extractTags())
-                   .withAlternativeAbstracts(Collections.emptyMap())
-                   .build();
+            .withLanguage(extractLanguage())
+            .withMainTitle(extractMainTitle())
+            .withPublicationDate(extractPublicationDate())
+            .withReference(new ReferenceBuilder(cristinObject).buildReference())
+            .withContributors(extractContributors())
+            .withNpiSubjectHeading(extractNpiSubjectHeading())
+            .withAbstract(extractAbstract())
+            .withTags(extractTags())
+            .withAlternativeAbstracts(Collections.emptyMap())
+            .withDescription(extractDescription())
+            .build();
+    }
+
+    private String extractDescription() {
+        return Optional.ofNullable(cristinObject.getCristinArtisticProduction())
```

Comment:
is CristinArtisticProduction the only one that uses description?

Reply:
So far yes, (and most likely to be so forever).

It all the fields from cristin that doesn't quite fit the NVA-datamodel. Not a design award this implementation, but it's either that or  dataloss.